frontend/src/views/web/challenge/manage.html

```diff
@@ -12,6 +12,11 @@ <h5 class="w-300">Submission worker actions</h5>
                 <button class="btn ev-btn-dark waves-effect waves-dark grad-btn grad-btn-dark fs-14" ng-click="challenge.manageWorker('restart');"><i class="fa fa-refresh"></i> Restart worker</button>
             </div>
         </div>
+        <div class="row margin-bottom-cancel">
+            <div class="col s12">
+                <h5 class="w-300">Submission worker status - {{challenge.workerStatus?challenge.workerStatus:"Loading..."}}</h5>
```

Comment:
@burnerlee can we should color coded status here. 

1. RUNNING - Green
2. PENDING - Orange
3. STOPPED - RED